opentelemetry-sdk/src/opentelemetry/sdk/_metrics/instrument.py

```diff
@@ -97,6 +97,10 @@ def callback(self) -> Iterable[Measurement]:
                     )
             except StopIteration:
                 pass
+            except Exception:  # pylint: disable=broad-except
+                _logger.warning(
```

Comment:
Just a suggestion, [exception](https://docs.python.org/3/library/logging.html#logging.Logger.exception) may be a better choice here.